File: Leetcode Challenge/February/cpp/Remove Duplicates from Sorted Array II.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int i=0;
        for(auto e:nums){
            
            if(i==0 || i==1 || nums[i-2]!=e){
                nums[i]=e;
                i+=1;
            }
        }
        
        return i;
    }
};

File: Leetcode Challenge/February/java/Remove Duplicates from Sorted Array II.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int i=0;
        for(int k=0;k<nums.length;k++){
            
            if(i==0 || i==1 || nums[i-2]!=nums[k]){
                nums[i]=nums[k];
                i+=1;
            }
        }
        
        return i;
    }
}
